binary-tiling.cpp

```diff
@@ -486,6 +486,8 @@ EX namespace bt {
             add(-2 * t0 + shift1);
             }
           }
+          // WARNING! UNINTENTIONAL FALLTHROUGH??
+          goto fallthru; fallthru:

```

Comment:
Leaving a review comment to call this out even more explicitly. I have no idea what this code is doing, but it kind of _looks_ like it could be a bug.